sensu/docker/my_init.d/ssl_generate.sh

```diff
@@ -0,0 +1,18 @@
+#!/bin/sh
```

Comment:
we are gonna generate new cert each time we start container - is it what you wanted?